drivers/net/wireless/mediatek/mt76/mt76_connac_mcu.c

```diff
@@ -875,10 +875,13 @@ int mt76_connac_mcu_add_sta_cmd(struct mt76_phy *phy,
 	if (IS_ERR(skb))
 		return PTR_ERR(skb);
 
-	mt76_connac_mcu_sta_basic_tlv(skb, info->vif, info->sta, info->enable);
-	if (info->enable && info->sta)
-		mt76_connac_mcu_sta_tlv(phy, skb, info->sta, info->vif,
-					info->rcpi);
+	if (info->sta) {
+		mt76_connac_mcu_sta_basic_tlv(skb, info->vif, info->sta,
```

Comment:
@objelf mt76_connac_mcu_add_sta_cmd() is run by mt7663 driver with info->sta = NULL and info->enable = NULL in mt7615_remove_interface(). Can you please double check the mt76_connac_mcu_sta_basic_tlv() is not necessary in this case (mt7663 device)?